generic-types-traits/includes/5-knowledge-check.md

```diff
@@ -0,0 +1,37 @@
+# Knowledge check
+
+What does the following type declaration means?
+
+```rust
```

Comment:
I meant to tell you this earlier, but the knowledge checks can't contain questions that have multiple lines. Is there a way to shorten these questions, or specifically the code, into one line?

Reply:
I can't think of viable code that could fit in a single line, but maybe we can use links to the Rust Playground to display the code. Do you think this would work?

If not, I can try to come up with questions in prose form. That won't be a problem at all.

Reply:
Yes, I think questions in prose form will work best. Sorry, I really like these type of questions but the platform won't allow it.

Reply:
No problem! =)